WooCommerce/Classes/ViewRelated/Upgrades/SubscriptionsViewModel.swift

```diff
@@ -80,10 +82,38 @@ final class SubscriptionsViewModel: ObservableObject {
     func loadPlan() {
         storePlanSynchronizer.reloadPlan()
     }
+
+    /// Opens the subscriptions management URL
+    ///
+    func onManageSubscriptionButtonTapped() {
+        if let url = URL(string: "https://apps.apple.com/account/subscriptions") {
```

Comment:
This could potentially be added to `WooConstants.URL`, which has an unwrap function too.

Reply:
Changed here: https://github.com/woocommerce/woocommerce-ios/pull/9985/commits/11095e6dbf4f3d8e4f73a7343135f5cbf6db7f31